Services/WordCount/WordCount.Service/Controllers/DefaultController.cs

```diff
@@ -50,9 +54,47 @@ public DefaultController(IReliableStateManager stateManager)
                 await queue.EnqueueAsync(tx, word);
 
                 await tx.CommitAsync();
+
+                await UpdateStats();
             }
 
             return this.Ok();
-        }
+        }
+
+        private async Task<long> UpdateStats(int count = 1)

```

Comment:
From my understanding you cannot call UpdateStats from AddWord because then you try to update the ReliableDictionary from activeSecondary too.